daily-patch-information/src/main/java/org/wso2/patchinformation/MainEmailSender.java

```diff
@@ -0,0 +1,132 @@
+//
+// Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
```

Comment:
You need to decide on the groupId and artefactId, and the package name should be reflecting those. For example, as this repo name engineering-efficiency, all the components in this repo will be having the common group Id. Therefore one possible group id  will be 'org.wso2.engineering.efficiency' and the artifact Id of this component may be 'org.wso2.engineering.efficiency.patch.analsis'. Please think through the package name in align with this, and refactor the source code.